processor/src/main/java/org/mapstruct/ap/spi/SuffixEnumTransformationStrategy.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.spi;
+
+/**
+ * @author Filip Hrisafov
+ */
+public class SuffixEnumTransformationStrategy implements EnumTransformationStrategy {
+
+    @Override
+    public String getType() {
```

Comment:
Maybe a different name then `type`? As far as I understood this has to be the same value than `nameTransformStrategy` in the new `EnumMapping` annotation?

Reply:
Yes this is the name / type of the `strategy` that you can reference in `nameTransformStrategy`. What would you propose? I of course need to add Javadoc

Reply:
```suggestion
    public String getStrategyName() {
```

Maybe?

Reply:
Good idea. Will apply it